### PR TITLE
Added a modal to notify user if a existing triage exist

### LINKED ISSUE #3950: Triage old data is getting replaced with new data in facility
**Describe the bug**

When we submit multiple triage data entry for a single date in the facility page, the last submitted one will be replacing the old data of the date

**To Reproduce**

https://care.coronasafe.in/facility/c4463366-0e21-45c0-9da3-25457e7413e5

**Expected behavior**

- [ ] when someone submitting a triage data, and if a data already exist for the specific date, a pop-up comes on submission stating "data will be replaced"

**Screenshots**

image

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes
Fixes #3950
Added a modal to notify user if a existing triage exist
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@patelaryan7751 the functionality looks good ! but, the text content seem bulky, shall we discuss it on EOD, to see other opinions too !

**Comment by nihal467** [comment]
- "a triage already exist in the date" can be on first row and move the remaining content to the next line and use smaller font size for that
- use the tailwind button
- use the danger or warning exist for the proceed button
- use secondary for the cancel button

**Comment by patelaryan7751** [comment]
Hey @nihal467 all the modal Ui changes are done.
image

**Comment by nihal467** [comment]
LGTM

File: src/Components/Common/components/ButtonV2.tsx [hunk 1]
```diff
@@ -82,6 +87,7 @@ const ButtonV2 = ({
         shadow && "shadow enabled:hover:shadow-lg",
         className,
       ].join(" ")}
+      id={`${id || ""}`}
```
Comment (khavinshankar) [inline src/Components/Common/components/ButtonV2.tsx]: why aren't you just doing
```suggestion
      id={id}
```

File: src/Components/Facility/TriageForm.tsx [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+import {
+  Button,
+  Card,
+  CardContent,
+  InputLabel,
+  Modal,
+} from "@material-ui/core";
+
+
+
```
Comment (khavinshankar) [inline src/Components/Facility/TriageForm.tsx:11]:
```suggestion
```

extra lines seems unnecessary

File: src/Components/Facility/TriageForm.tsx [hunk 3]
```diff
@@ -1,4 +1,14 @@
-import { Card, CardContent, InputLabel } from "@material-ui/core";
+
```
Comment (khavinshankar) [inline src/Components/Facility/TriageForm.tsx:1]:
```suggestion
```
extra lines seems unnecessary

File: src/Components/Facility/TriageForm.tsx [hunk 4]
```diff
@@ -9,12 +19,17 @@ import {
...
   getAnyFacility,
+  getTriageInfo,
 } from "../../Redux/actions";
 import * as Notification from "../../Utils/Notifications.js";
 import { TextInputField } from "../Common/HelperInputFields";
 import { PatientStatsModel } from "./models";
 import { goBack } from "../../Utils/utils";
+
+import ButtonV2 from "../Common/components/ButtonV2";
+
```
Comment (khavinshankar) [inline src/Components/Facility/TriageForm.tsx]:
```suggestion
import ButtonV2 from "../Common/components/ButtonV2";
```

**Review by khavinshankar — CHANGES_REQUESTED** [review]
@patelaryan7751 some minor changes

**Comment by nihal467** [comment]
@patelaryan7751 can you work on it and push it after fixing all the problems before new year

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**